concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md

```diff
@@ -18,4 +18,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
 - **Editing column**        
 A type of buttons column. Contains editing controls. See [Customize the Editing Column](/concepts/05%20Widgets/DataGrid/15%20Columns/10%20Column%20Types/4%20Command%20Columns/15%20Customize%20the%20Editing%20Column/0%20Customize%20the%20Editing%20Column.md '/Documentation/Guide/Widgets/DataGrid/Columns/Column_Types/Command_Columns/#Customize_the_Editing_Column').
```

Comment:
A type of buttons column. --> This phrasing doesn't make sense. Can you please explain what you are trying to say? Are you saying that this is a column of different button types?

Reply:
That's not me (my changes are marked with green). This is an old text written long ago.

The point is the following: there are several types of buttons columns - editing, custom and others.